### PR TITLE
Fixes #14125 - Updating katello-remove cleanup

### PR DESCRIPTION
**Pull request by chris1984**
Updating katello-remove to clean up orhpaned directories

### CODE REVIEW

File: katello/katello-remove [hunk 1]
```diff
@@ -6,7 +6,7 @@ echo "Important packages such as the following will be removed:"
 echo ""
 echo "  * httpd (apache)"
 echo "  * mongodb"
-echo "  * tomcat6"
+echo "  * tomcat"
```
Comment (beav) [inline katello/katello-remove:16]: technically it is still `tomcat6` on rhel 6, but I think this update is more correct :cat2:

**Comment by beav** [comment]
@chris1984 which part removes /etc/gutterball? the `rpm -qa | grep tomcat` ?

File: katello/katello-remove [hunk 2]
```diff
@@ -103,13 +103,12 @@ RPMS=(
     `rpm -qa | grep ^python-isodate`
     `rpm -qa | grep gutterball`
```
Comment (chris1984) [inline katello/katello-remove:113]: @beav gutterball is already being removed in katello-remove I have modified katello-backup which is coming in a new PR

**Comment by chris1984** [comment]
@beav still looking at a few things, please hold of on a merge till I give the thumbs up on my side.

File: katello/katello-remove [hunk 3]
```diff
@@ -124,6 +123,14 @@ CERT_FILES=(
     /var/www/html/pub/katello-ca*.rpm
 )
 
+CONTENT=(
+    /var/lib/qpid
+    /var/lib/hiera
+    /usr/share/qpid
+    /usr/share/qpid-tools
+    /usr/share/katello-installer
```
Comment (ehelms) [inline katello/katello-remove]: I think this is `/usr/share/katello-installer-base`, there will also be `/usr/share/foreman-installer`. There is also now `/etc/foreman-installer` for Katello 3.0  @mbacovsky any I am missing?

Comment (mccun934) [inline katello/katello-remove]: +1 to /etc/foreman-installer as well as:

```
/var/log/foreman-installer/
/var/www/html/pub/katello-server-ca.crt
/usr/share/foreman-installer-katello
/etc/sudoers.d/foreman-proxy
/etc/hammer/
/usr/share/foreman-proxy
/var/cache/pulp
```

**Comment by ehelms** [comment]
@chris1984 looks like there are still some comments to address for this PR

**Comment by chris1984** [comment]
@ehelms thanks yep still working on it, that was Brad helping me trying to get past a troublesome rebase, should have this addressed this week.

**Comment by mccun934** [comment]
ACK - removed and re-installed without issue

**Comment by ehelms** [comment]
Thanks @chris1984 !

**Pull request merged**